src/structs/field-shorthand.md

```diff
@@ -32,6 +32,20 @@ impl Person {
         Self { name, age }
     }
 }
+```    
+* Implement a `default` setting for the struct.
+```rust,ignore
+impl Default for Person {
+  fn default() -> Person { Person { name:"Bot".to_string(), age:0 } }
+}
+```
+* Create a new struct by using the default values.
+```rust,ignore
+let tmp = Person { ..Default::default() };
+```
+* Define some fields and use the default values for the other fields.
+```rust,ignore
```

Comment:
The code blocks are not indented correctly so they don't become part of the list items:

![image](https://user-images.githubusercontent.com/89623/222768363-e39d6172-7679-4530-8981-fcc4b1fd211e.png)

You can actually see this as well in the rich diff shown in GitHub:

![image](https://user-images.githubusercontent.com/89623/222768757-b43a6876-ada7-4ea4-a6bb-21bc2d83d9d4.png)

Reply:
Noticed this wasn't done consistently and opened issue #475